model/config.go

```diff
@@ -1456,7 +1456,7 @@ func (s *EmailSettings) SetDefaults(isUpdate bool) {
 	}
 
 	if s.PushNotificationContents == nil {
-		s.PushNotificationContents = NewString(GENERIC_NOTIFICATION)
+		s.PushNotificationContents = NewString(FULL_NOTIFICATION)
```

Comment:
This change does not appear in the changelog...

Reply:
That seems like it might've been an oversight, but it would only affect new install of Mattermost. These defaults only come into affect if that setting is completely missing from the config.

Reply:
You are right. My deployment is a big unusual (default config, gets populated with values I set, so anything I didn't set explicitly keeps its default). I should probably change that at some point :)

Reply:
cc// @amyblais re this being missed in the changelog ^

Reply:
Ah, I see. Assuming you have some way to persist the config.json, you could consider setting any specific values you want using environment variables if you want to configure them from outside Mattermost. I think there's also an option to store the configuration in the database, but I haven't tried that out myself.